### PR TITLE
certificate usage clarification

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#4861

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix Copy editing is needed for this PR. Thanks.

File: windows/security/information-protection/windows-information-protection/create-and-verify-an-efs-dra-certificate.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ The recovery process included in this topic only works for desktop devices. WIP
 
 4. Add your EFS DRA certificate to your WIP policy using a deployment tool, such as [Microsoft Intune](create-wip-policy-using-intune-azure.md) or [System Center Configuration Manager](create-wip-policy-using-sccm.md).
 
+>[!NOTE]
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/create-and-verify-an-efs-dra-certificate.md]:
```suggestion
> [!NOTE]
```

File: windows/security/information-protection/windows-information-protection/create-and-verify-an-efs-dra-certificate.md [hunk 2]
```diff
@@ -49,6 +49,9 @@ The recovery process included in this topic only works for desktop devices. WIP
...
+>This certificate could be used in Intune for both policies with device enrollment (MDM) and without device enrollment (MAM).
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/create-and-verify-an-efs-dra-certificate.md]:
```suggestion
> This certificate can be used in Intune for policies both _with_ device enrollment (MDM) and _without_ device enrollment (MAM).
```

Comment (illfated) [reply windows/security/information-protection/windows-information-protection/create-and-verify-an-efs-dra-certificate.md]: (The sentence can likely be written differently and still keep its original meaning. I just went with the version I felt is more obvious.)

**Comment by e0i** [comment]
cc @MaratMussabekov

**Comment by MaratMussabekov** [comment]
Dear @illfated ,
your recommendations were applied
Thank you!

**Comment by e0i** [comment]
@dulcemontemayor The PR that addresses issue #4861 is available for review. Thanks.

**Comment by e0i** [comment]
@DulceMontemayor Following up for updates.

cc @Dansimp

**Comment by e0i** [comment]
Still under escalation.

**Pull request merged**